src/test/java/uk/gov/justice/hmpps/prison/aop/connectionproxy/OracleConnectionAspectTest.kt

```diff
@@ -169,18 +151,14 @@ class OracleConnectionAspectTest {
 
         connectionAspect.openProxySessionIfIdentifiedAuthentication(pooledConnection)
 
-        verify(pooledConnection).prepareStatement(
-          check {
-            assertThat(it).contains("nomis_context.set_context('AUDIT_MODULE_NAME', 'MERGE')")
-          }
-        )
+        verify(nomisConfigurer).setNomisContext(any<ResettableContextConnection>(), eq("some user name"), eq("some IP"), eq("some URI"), eq("APP"), eq(true))
       }
 
       @Test
-      fun `should return original connection`() {
+      fun `should return resettable context connection`() {
         val conn = connectionAspect.openProxySessionIfIdentifiedAuthentication(pooledConnection)
 
-        assertThat(conn).isEqualTo(pooledConnection)
+        assertThat(conn).isExactlyInstanceOf(ResettableContextConnection::class.java)
```

Comment:
And we are also returning a resettable context connection where we set the NOMIS context without an Oracle proxy connection.